src/components/app/App.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { PrivateRoutes, PublicRoutes } from '../../router/Router';
+
+import './App.scss';
+
+function App() {
+  const login: boolean = true;
+  return (
+    <section className="wrapper">
+      {/*{login ? <PrivateRoutes /> : <PublicRoutes />}*/}
```

Comment:
если хочешь оставить какой-то код закомментированным, то добавь текстовое описание, что это и зачем
```suggestion
      // Пояснение тут
      {/*{login ? <PrivateRoutes /> : <PublicRoutes />}*/}
```